dali/pipeline/data/tensor_vector.cc

```diff
@@ -395,18 +389,21 @@ std::shared_ptr<TensorList<Backend>> TensorVector<Backend>::AsTensorList(bool ch
 
 template <typename Backend>
 void TensorVector<Backend>::resize_tensors(int new_size) {
-  auto old_size = tensors_.size();
-  tensors_.resize(new_size);
-  for (int i = old_size; i < new_size; i++) {
-    tensors_[i] = std::move(std::make_shared<Tensor<Backend>>());
-    tensors_[i]->set_pinned(pinned_);
+  if (new_size > tensors_.size()) {
```

Comment:
signed/unsigned comparison - will cause an error (-Werror) in clang
```suggestion
  if (new_size > static_cast<int>(tensors_.size()) {
```

Reply:
Done